app/controllers/admin/api/services/proxies_controller.rb

```diff
@@ -63,13 +63,7 @@ def show
   ##~ op.parameters.add name: "jwt_claim_with_client_id_type", description: "JWT Claim With ClientId Type. Either `plain` or `liquid`", dataType: "string", paramType: "query", required: false
   #
   def update
-    if proxy.update_attributes(proxy_params)
-      if proxy.service_mesh_integration?
-        proxy.deploy!
-      elsif proxy.apicast_configuration_driven
-        proxy.deploy_v2
-      end
-    end
+    proxy.deploy_mesh_or_apicast if proxy.update_attributes(proxy_params)
```

Comment:
There is no method `deploy_mesh_or_apicast`

Reply:
Right :joy: Good catch :+1: